test/sol5/kyberMatchingEngine.js

```diff
@@ -113,18 +114,18 @@ contract('KyberMatchingEngine', function(accounts) {
 
         it("should not have unauthorized personnel set fee accounted data", async() => {
             await expectRevert(
-                matchingEngine.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: operator}),
+                storage.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: operator}),
                 "only admin"
             );
 
             await expectRevert(
-                matchingEngine.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: network}),
+                storage.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: network}),
```

Comment:
please don't test this in matchingEngine test file.

move to kyberStorage test file.

Reply:
moved